### PR TITLE
README: add description how to use git clone as local repository

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,11 @@
 [![Crowdin Badge](https://d322cqt584bo4o.cloudfront.net/phoenicis-scripts/localized.svg)](https://crowdin.com/project/phoenicis-scripts)
 
 # How to add a script
-1. Select the right category
+1. Clone this repository
+
+2. Add the local checkout as local repository in Phoenicis (see [instructions](https://phoenicisorg.github.io/phoenicis/Users/repository/#local-directory))
```
Comment (madoar) [inline README.md]: `... as a local repository ...`

File: README.md [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+
+3. Select the right category
```
Comment (madoar) [inline README.md]: How do you select the category?

File: README.md [hunk 3]
```diff
@@ -3,7 +3,11 @@
 [![Crowdin Badge](https://d322cqt584bo4o.cloudfront.net/phoenicis-scripts/localized.svg)](https://crowdin.com/project/phoenicis-scripts)
 
 # How to add a script
-1. Select the right category
+1. Clone this repository
```
Comment (madoar) [inline README.md]: Maybe also add the github link, which should be cloned

File: README.md (hunk 2 above)
Comment (plata) [reply README.md]: Well, where does the script belong to.

Comment (madoar) [reply README.md]: Yes I know that, but normally when I read or hear `select` I think about clicking on it, which is not what you mean here.
I think you want to say something like:
```
3. Select the right category through the use of its path in the next steps
```

Comment (plata) [reply README.md]: I get your point but I don't think this makes it clearer.

**Comment by madoar** [comment]
@plata it seems like the indentation of the PR is not right

**Comment by plata** [comment]
Why?

**Pull request merged**